### PR TITLE
Feat/data source list and Cloudfeed renaming. Also rename Twomes to NeedForHeat and removes Building

### PR DESCRIPTION
**Pull request by Labhatorian**
This PR expands the database and API with items for the DataSourceList whilst also renaming cloudfeed to what had been discussed earlier and fixing the longitude typo.

This will likely break the existing app and we probably need to rename API's and models in the app before we can connect this. That's why I recommend putting this version on a seperate endpoint so we can ensure everything is working correctly.

Apologies for the big PR, there are a lot of files that needed to be added and changed. Next time the renames should be done separately

Update: Now removes Building as well

### CODE REVIEW

**Review by n-vr — COMMENTED** [review]
See comments

**Review by n-vr — COMMENTED** [review]
See comments

File: services/devicetype.go
Comment (Labhatorian) [reply services/devicetype.go:1]: I believe it was because I was having trouble with it at first. I've set up the association, tested it and it works now, but we do have `TypeInstanceType` now which isn't pretty.

File: repositories/datasourcetype.go
Comment (Labhatorian) [reply repositories/datasourcetype.go:25]: Its purpose is so we can return the DataSourceType and its Order with the DataSourceList it has a relation with. The Order field is actually in `data_source_list_items` which is the many2many table.

File: README.md
Comment (henriterhofte) [inline README.md:46]: why the '-local' part in the name? this seems to deviate from the pattern

File: handlers/account.go
Comment (henriterhofte) [inline handlers/account.go:72]: Latitude and Longitude were stored with the building, which was originally created in the db at the time of account activation. With the new EnergyQuery facility, however, lat and lon are no longer in the database as separate fields. I expect that, if a campaign needs to know the building's timezone, this data will be have to be uploaded as part of the response to an EnergyQuery (e.g. the BuildingProfile EnergyQuery seems particularly well suited for this). 

So.. perhaps skip these parameters here?

One other thing to look out for: usage of ISO8601 fully qualified time stamps in response values of the API. In the v2 API, we already started accepting unix time timestamps in uploads and measurements.

In some API responses, however, we always used ISO8601 fully qualified time (including timezone offset) as timestamps. This was based on the assumption that all things related to accounts, via building(s) could always look at the timezone associated with the building and then onstruct a fully qualified ISO8601 timestamp, even if only a unix timewould have been available. When we ditch the manadatory TZName, this assumption no longer holds.

As a consequence, in the API v3 we may now fully switch to unix time for timestamps in the API. Any process that requires knowing or displaying a local time should get the TZName separately (analysis: from an uploaded EnergyQuery; NeedForHeat GearUp app (e.g. when displaying timestamps in the datasouces list: based on the timezone stored in the smartphone). 

In particular, for the API this means:
- any timestamp in the response now also should be specified in unix time only. 
- it may be wise to stop accepting ISO8601 timestamps in requests, only support Unix times.

File: needforheat/building/building.go
Comment (henriterhofte) [inline needforheat/building/building.go:1]: can this file be ditched? seems deprecated now...

File: repositories/building.go
Comment (henriterhofte) [inline repositories/building.go:1]: can this file be deleted? we ditch building?

File: services/account.go
Comment (henriterhofte) [inline services/account.go:98]: ditch lat, lon and tzname? no longer part of account

File: services/account.go
Comment (henriterhofte) [inline services/account.go:115]: ditch lat, lon and tzname? no longer part of account

**Review by henriterhofte — COMMENTED** [review]
Three main points to look at (see detiled comments)
- can/should you already remove anything related to builgdings?
- can/shoud you remove lat, lon and TZname [parameters from the account creation API?
- implications for timestamps in API: v3 should only support Unix time now? (leave it to analysis and smartphone app themselfves to get a proper TZName to display/interpret the Unix times in terms of local time)

File: handlers/account.go
Comment (Labhatorian) [reply handlers/account.go]: Seems to cause no issues? Of course we'd need to test this when this goes live on testing, but looking into the code there were only a few instances of `.toISOString()` which I changed to `.toLocaleString()` which (according to chatgpt) should use the device's timezone to convert.

I've done this change which was quick and easy but this would likely mean that WeatherInterpolationLocation should be done first before anything else in the DataSourceList most likely.

Comment (henriterhofte) [reply handlers/account.go]: I only see .toLocaleString() in the Swagger .js files. 
In the JSON requests and responses themselves, only Unix time should be supported in API v3. 
I am not sure that .toLocateString() achieves what we want. 

Shoudn't you use something like:

 unixTimeInSeconds = Math.floor(someDateTime.getTime() / 1000);

Nevertheless, I will approve the PR, so you can check.

**Review by henriterhofte — APPROVED** [review]
There's one conversation regarding use of .toLocaleString() that I marked as resolved , but yuo may want to pay some attention to during testing.

**Comment by Labhatorian** [comment]
@henriterhofte I can't merge it because it says I am not authorized. Could you do it or give me permission? It'll be easier to push out hotfixes if we can do that ourselves

**Pull request merged**
